.gitignore

```diff
@@ -21,4 +21,51 @@ source/patterns/**/tsconfig.json
 deployment/dist/*
 .DS_Store
 
-*.pptx
\ No newline at end of file
+*.pptx
```

Comment:
Please remove/rollback these changes, it's not needed

Reply:
removing all the package.json entries